<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>486</width>
                <height>296</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>BBoxToolbox</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
            <item row="1" column="1">
                <widget class="QgsMapLayerComboBox" name="destinationLayer">
                    <property name="allowEmptyLayer">
                        <bool>true</bool>
                    </property>
                    <property name="showCrs">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>Output Layer:</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="0">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>BBox format:</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="1">
                <widget class="QComboBox" name="formatsComboBox"/>
            </item>
            <item row="4" column="0">
                <widget class="QLabel" name="label_3">
                    <property name="text">
                        <string>BBox:</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="1">
                <widget class="QPlainTextEdit" name="bboxString"/>
            </item>
            <item row="5" column="1">
                <widget class="QPushButton" name="showButton">
                    <property name="text">
                        <string>Show</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="QgsProjectionSelectionWidget" name="projectionSelectionWidget"/>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                    <property name="text">
                        <string>Projection:</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QgsMapLayerComboBox</class>
            <extends>QComboBox</extends>
            <header>qgsmaplayercombobox.h</header>
        </customwidget>
        <customwidget>
            <class>QgsProjectionSelectionWidget</class>
            <extends>QWidget</extends>
            <header>qgsprojectionselectionwidget.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
